Makefile

```diff
@@ -146,12 +146,21 @@ TEST_APK_PROJECTS = \
 	tests/CodeGen-Binding/Xamarin.Android.JcwGen-Tests/Xamarin.Android.JcwGen-Tests.csproj \
 	tests/locales/Xamarin.Android.Locale-Tests/Xamarin.Android.Locale-Tests.csproj
 
+TEST_APK_PROJECTS_RELEASE = \
+	src/Mono.Android/Test/Mono.Android-Tests.csproj
+
 # Syntax: $(call BUILD_TEST_APK,path/to/project.csproj)
 define BUILD_TEST_APK
 	# Must use xabuild to ensure correct assemblies are resolved
-	MSBUILD="$(MSBUILD)" tools/scripts/xabuild $(MSBUILD_FLAGS) /t:SignAndroidPackage $(1)
+	MSBUILD="$(MSBUILD)" tools/scripts/xabuild $(MSBUILD_FLAGS) /t:SignAndroidPackage $(2) $(1)
 endef	# BUILD_TEST_APK
 
+define RUN_APK_TESTS
+	$(foreach p, $(1), $(call BUILD_TEST_APK, $(p),$(2))); \
```

Comment:
This shouldn't use `; \`; I think that will silence any errors which are reported.